test/cli_create_test.go

```diff
@@ -193,6 +193,25 @@ func (suite *PouchCreateSuite) TestCreateWithCapability(c *check.C) {
 	}
 }
 
+// TestCreateWithPrivilege tries to test create a container with privilege.
+func (suite *PouchCreateSuite) TestCreateWithPrivilege(c *check.C) {
+	name := "create-privilege"
+
+	res := command.PouchRun("create", "--name", name, "--privileged", busyboxImage, "brctl", "addbr", "foobar")
+	res.Assert(c, icmd.Success)
+
+	output := command.PouchRun("inspect", name).Stdout()
+
+	result := &types.ContainerJSON{}
+	if err := json.Unmarshal([]byte(output), result); err != nil {
+		c.Errorf("failed to decode inspect output: %v", err)
+	}
+
+	if !result.HostConfig.Privileged {
```

Comment:
Here we can use assert to simplify the test case.

We usually use assert in test.